packages/renderer/src/lib/kube/KubeApplyYAMLButton.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+import Button from '../ui/Button.svelte';
+import type { KubernetesObject } from '@kubernetes/client-node';
+import SolidKubeIcon from '../images/SolidKubeIcon.svelte';
+
+let inProgress = false;
+
+async function kubeApply(): Promise<void> {
+  let contextName = await window.kubernetesGetCurrentContextName();
+  if (!contextName) {
+    return;
+  }
+
+  const result = await window.openFileDialog('Select a .yaml file to apply', {
+    name: 'YAML files',
+    extensions: ['yaml', 'yml'],
+  });
+  if (result.canceled || result.filePaths.length !== 1) {
+    return;
+  }
+
+  let file = result.filePaths[0];
+  if (!file) {
+    return;
+  }
+
+  inProgress = true;
+  try {
+    let objects: KubernetesObject[] = await window.kubernetesApplyResourcesFromFile(contextName, file);
+    await window.showMessageBox({
+      title: 'Kubernetes',
+      type: 'info',
+      message: `${objects.length} resource(s) applied`,
```

Comment:
Could we say something like: 'Successfully applied {objects.length} resource(s)'?

Just to indicate it's successful :)

Reply:
Fixed.